charts/rancher-vsphere-cpi/templates/configmap.yaml

```diff
@@ -19,13 +19,11 @@ data:
     vcenter:
       {{ .host | quote }}:
         server: {{ .host | quote }}
```

Comment:
Is it OK to leave `server` here with the same value used as the section itself (`.host`)?

Reply:
I believe so since this is how upstream has it configured https://github.com/kubernetes/cloud-provider-vsphere/blob/06570c94f4c93688feb5c995ade3e384c9086b15/charts/vsphere-cpi/templates/configmap.yaml#L28-L29.